### DISCUSSION TITLE
可以增加字体设置的选项吗

### DISCUSSION THREAD

**Discussion by Sunsh4j** [tw93/Pake#412] [Ideas]
安装后，打开应用 显示的字体有点糊啊，如下图：
![image](https://user-images.githubusercontent.com/43169575/235709016-98918b80-39cf-476c-aa6f-591e86557e45.png)
在网页端，我在chrome里设置了别的字体，显示的效果就好些了，如下图：
![image](https://user-images.githubusercontent.com/43169575/235709079-0c0b6054-4476-4fd3-ac1a-ac815adf45cf.png)
所以，如果可以的话，希望可以增加设置字体的选项🧡🧡🧡

**Comment by tw93** [comment]
懂你说的了，这样属于系统的限制了，看看你的系统浏览器默认字体是不是有问题，可以重新安装下那个字体试试

**Reply by Sunsh4j** [reply]
字体就是微软雅黑，但是我觉得在windows的低分屏上显示效果不好，所以安装了别的字体，然后用noMeiryoUI指定了全局字体，显示效果还可以，但是有极少数软件还是用的默认。这个"糊"的情况倒是第一次遇见🤣🤣
